medusa/indexers/tvmaze/tvmaze_api.py

```diff
@@ -371,7 +371,7 @@ def _parse_actors(self, tvmaze_id):
         for order, cur_actor in enumerate(actors.people):
             save_actor = Actor()
             save_actor['id'] = cur_actor.id
-            save_actor['image'] = cur_actor.image.get('original')
+            save_actor['image'] = cur_actor.image.get('original') if cur_actor.image else ''
```

Comment:
None of your alternatives work. Only alternative could be: ```getattr(cur_actor, 'image', {}).get('originall', '')```

Ben then I couldn't say if that would be any better then what I currently have.